### ISSUE TITLE
Great Plugin not running anny more with Illustrator 25.0 - empty panel

### ISSUE THREAD

**Issue by hirschferkel** [majman/adobe-scripts-panel#11]
Is there a way to fix this in Illustrator 25.0? The Script Panel is empty, now...
<img width="488" alt="Bildschirmfoto 2020-10-21 um 09 54 46" src="https://user-images.githubusercontent.com/16016422/96690131-98322a00-1383-11eb-83e7-db540726a266.png">

**Comment by daani-rika** [comment]
Did you set PlayerDebugMode to 1 like readme said?

**Comment by hirschferkel** [comment]
> Did you set PlayerDebugMode to 1 like readme said?

Of course, otherwise it would not work in Illustrator 2020. Everything fine, there. But it does not work in Illustrator 2021?

**Comment by Klustre** [comment]
Did you set debug mode for CEP 10? `defaults write com.adobe.CSXS.10 PlayerDebugMode 1`

**Comment by hirschferkel** [comment]
Oh you're great, that's it! Great... thanx a lot
